code/modules/mob/living/basic/icemoon/ice_whelp/ice_whelp_abilities.dm

```diff
@@ -0,0 +1,35 @@
+/datum/action/cooldown/mob_cooldown/ice_breathe
+	name = "Ice breathe"
+	desc = "Fire a cold line of fire towards the enemy!"
+	button_icon = 'icons/effects/magic.dmi'
+	button_icon_state = "fireball"
+	cooldown_time = 2 SECONDS
```

Comment:
This cooldown seems very short, does this not mean they're going to spend the majority of their time breathing fire?
It will at least have a significant effect on player-controlled dragons, so please make it a bit longer.